controller/src/beerocks/master/db/db.cpp

```diff
@@ -2289,6 +2290,24 @@ int db::get_channel_scan_dwell_time_msec(const sMacAddr &mac, bool single_scan)
         .dwell_time_msec;
 }
 
+bool db::is_channel_scan_pool_supported(const sMacAddr &mac,
+                                        const std::unordered_set<uint8_t> &channel_pool)
+{
+    auto supported_channels = get_hostap_supported_channels(network_utils::mac_to_string(mac));
+    for (const auto &channel : channel_pool) {
+        auto found_channel =
+            std::find_if(supported_channels.begin(), supported_channels.end(),
+                         [&channel](const beerocks::message::sWifiChannel &supported_channel) {
+                             return supported_channel.channel == channel;
+                         });
+        if (found_channel == supported_channels.end()) {
+            LOG(ERROR) << "channel #" << int(channel) << " is not supported";
```

Comment:
I completely understand
And I think I'll change it to `INFO`
But I still want the log, that way there is a log of which sent channel isn't supported
Since the function doesn't return the unsupported channel, the only way to know which is via the log